src/components/AddMovie/AddMovie.jsx

```diff
@@ -24,16 +19,14 @@ const AddMovie = (props) => {
 
   const handleSubmit = (event) => {
     event.preventDefault();
-    handleAdd(film);
+    setFilms(film);
   };
 
   const handleInputChange = (event) => {
     event.preventDefault();
     setFilm({
       ...film,
       [event.target.name]: event.target.value,
-      id: len + 1,

```

Comment:
this logic is moved to src/components/SearchResultsPage/SearchResultsPage.jsx  handleSetFilm